tests/rptest/services/redpanda.py

```diff
@@ -1694,6 +1695,9 @@ def unset_environment(self, keys: list):
             except KeyError:
                 pass
 
+    def add_permitted_anomaly(self, anomaly: str):
+        self._additional_permitted_anomalies.add(anomaly)
+
```

Comment:
This can already be achieved at runtime via `RedpandaService.SISettings.set_expected_damage`. Granted, the semantics are a bit different: `set_expected_damage` silences a fatal anomaly and `add_permitted_anomaly` makes the anomaly non-fatal.